master/internal/rm/db.go

```diff
@@ -0,0 +1,49 @@
+package rm
+
+import (
+	"context"
+	"time"
+
+	"github.com/determined-ai/determined/master/internal/db"
+	"github.com/determined-ai/determined/master/pkg/model"
+	"github.com/determined-ai/determined/proto/pkg/jobv1"
+)
+
+// FetchAvgQueuedTime fetches the average queued time for a resource pool.
+func FetchAvgQueuedTime(pool string) (
```

Comment:
are there any Go practices for keeping the exposure limited this and subpackages aka `rm/*` ?